apps/tailwind-components/pages/input/String.story.vue

```diff
@@ -1,94 +1,20 @@
 <script setup lang="ts">
-const label = "Search for patients";
-const placeholder = "'A-001' or 'A-001,A-002'";
-const demoValue = ref("");
+const demoValue = ref("test");
 </script>
 
 <template>
-  <h2>String component</h2>
-  <p>
-    The <code>String Component</code> enables you to create an input element
```

Comment:
is it okay if I delete these long stories? They look nice and expensive but I must confess I find them a bit distracting.